projects/igniteui-angular/src/lib/grids/tree-grid/tree-grid.component.html

```diff
@@ -20,14 +20,16 @@
     </select>
 </ng-template>
 
-<div class="igx-grid__thead">
+<div class="igx-grid__thead" [style.width.px]='outerWidth'>
```

Comment:
@kdinev Because of IE.  In IE if the containers don't have explicit widths their content overflows. With the introduction of the external scrollbar we added new containers (for header, tbody and footer), which didn't have explicit widths, hence sizing was broken in IE (for all grids). This sets widths for the parent containers so that everything sizes correctly in IE.